jobs/JGLOBAL_PREP_OCEAN_OBS

```diff
@@ -20,7 +20,7 @@ export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush:${PYTHONPATH}
 ###############################################################
 # Run relevant script
 
-EXSCRIPT=${GDASPREPOCNOBSPY:-${HOMEgfs}/ush/exglobal_prep_ocean_obs.py}
+EXSCRIPT=${GDASPREPOCNOBSPY:-${HOMEgfs}/scripts/exglobal_prep_ocean_obs.py}
```

Comment:
I don't think this script is available as it is being linked in `ush`.  See [here](https://github.com/ShastriPaturi/global-workflow/blob/cb8e2256226179a5c8ca3a25e84ed44e9aeabdd2/sorc/link_workflow.sh#L211)